jdaviz/configs/imviz/tests/test_simple_aper_phot.py

```diff
@@ -20,9 +20,9 @@ def test_plugin_wcs_dithered(self):
         # Make sure invalid Data/Subset selection does not crash plugin.
         phot_plugin.data_selected = 'no_such_data'
         assert phot_plugin._selected_data is None
-        phot_plugin.subset_selected = 'no_such_subset'
+        phot_plugin._subset_selected_changed(event={'new': 'no_such_subset'})

```

Comment:
The `observe` doesn't work for subset and background anymore, so I have to change this. I really don't understand why.